photutils/centroids/gaussian.py

```diff
@@ -105,11 +105,13 @@ def centroid_1dg(data, error=None, mask=None):
 
     xy_data = [np.ma.sum(data, axis=i).data for i in (0, 1)]
 
+    # would need to use a different fitter if parameter bounds are used
+    fitter = LMLSQFitter()
```

Comment:
@larrybradley - gaussians have bounds enabled by default (on stddev) so if you are fitting a Gaussian I think it would be best to use TRFLSQFitter or another one, not ``LMLSQFitter``

Reply:
@astrofrog Are you sure about that?  Where are the bounds defined?  I don't see any bounds defined in `functional_models`?  Do you mean constrained to be postive?  I still get negative stddev's sometimes.....

Reply:
I've also noticed that `TRFLSQFitter` is about 4-5% slower than `LevMarLSQFitter` even without any bounds.

Reply:
@larrybradley yes the fact the stddev is required to be positive - it's formally treated as a bound.

Reply:
Do you see negative stddev values with LMLSQ or TRFLSQ or both?